### PR TITLE
try to add some info to the organizer guide

### PR DESCRIPTION
**Pull request by kweav**
Filled out the organizer guide a bit more with info on the goals and the schedule. Please leave suggestions for improvements!

### CODE REVIEW

File: organizer-guide.Rmd [hunk 1]
```diff
@@ -7,20 +7,50 @@ This event is a key event for professionals in the genomics community interested
...
 
-- Parallel tracks, 5+ hours across Nov 12/13
-- Scientific Lead and Co-Lead
-- ~4-10 participants per topic
+- 3 Parallel tracks, 5+ hours across Nov 12/13
+  - Scalable Machine Learning and Data Science
+  - Polygenic Risk Score Analysis
+  - Wrap a Tool (+ Intro AnVIL/WDL)
+- Scientific Lead and AnVIL Co-Lead for support
+- We expect perhaps 4-10 participants per track
```
Comment (avahoffman) [inline organizer-guide.Rmd]:
```suggestion
- We expect 4-10 participants per track
```

File: organizer-guide.Rmd [hunk 2]
```diff
@@ -7,20 +7,50 @@ This event is a key event for professionals in the genomics community interested
...
 ## Logistics {- .unlisted}
 
-- Parallel tracks, 5+ hours across Nov 12/13
-- Scientific Lead and Co-Lead
-- ~4-10 participants per topic
+- 3 Parallel tracks, 5+ hours across Nov 12/13
+  - Scalable Machine Learning and Data Science
+  - Polygenic Risk Score Analysis
+  - Wrap a Tool (+ Intro AnVIL/WDL)
+- Scientific Lead and AnVIL Co-Lead for support
```
Comment (avahoffman) [inline organizer-guide.Rmd]:
```suggestion
- Scientific Lead and AnVIL Outreach Co-Lead for support
```

File: organizer-guide.Rmd [hunk 3]
```diff
@@ -7,20 +7,50 @@ This event is a key event for professionals in the genomics community interested
...
+- We expect perhaps 4-10 participants per track
+
+## Example Schedule for a Track {- .unlisted}
+
+- First Day (2.5 hrs scheduled):
+  - Problem statement (perhaps with a short demo) by scientific lead
```
Comment (avahoffman) [inline organizer-guide.Rmd]:
```suggestion
  - Problem statement by scientific lead
```

File: organizer-guide.Rmd [hunk 4]
```diff
@@ -7,20 +7,50 @@ This event is a key event for professionals in the genomics community interested
...
+
+    This part of the session will provide background info on the track's overall topic and may include a brief summary of the pre-worked ideas. Perhaps 20-30 minutes?
```
Comment (avahoffman) [inline organizer-guide.Rmd]:
```suggestion
    This part of the session will provide background info on the track's overall topic and may include a brief summary of the pre-worked ideas. [Suggested time: 20-30 minutes]
```

File: organizer-guide.Rmd [hunk 5]
```diff
@@ -7,20 +7,50 @@ This event is a key event for professionals in the genomics community interested
...
+
+    What project will the track tackle? This should be quicker then the problem statement, focusing on understanding the participant backgrounds and skills and matching those with track ideas and actionable tasks to accomplish the goal.
+
+  - Onboarding by AnVIL co-lead
+
+    Handle billing accounts and other logistics necessary to accomplish the work.
+
+  - Work-regroup, work-regroup
+
+    Probably half an hour to an hour of formal work time left on this first day before dinner. Groups may choose to work together, informally after dinner.
```
Comment (avahoffman) [inline organizer-guide.Rmd]: Would love for this to be in table format!

**Review by avahoffman — APPROVED** [review]
Minor changes suggested! I think a table would look nice - maybe a table for day 1 and a separate table for day 2?

Also need to address the spelling items in the dictionary.txt file.

**Comment by ehumph** [comment]
@kweav sorry, I accidentally committed an edit instead of leaving it a suggestion. Feel free to revert.

LINE 10
Changed:
- Build a community (inclusive to newcomers)

Now reads:
- Build an inclusive community that is supportive of newcomers

LINE 12
Changed:
- Report out a final product that accomplishes one of the following for a topic

Now reads:
- Produce a final product that accomplishes one of the following for a topic

File: organizer-guide.Rmd (hunk 5 above)
Comment (ehumph) [reply organizer-guide.Rmd]: I like the table idea

File: organizer-guide.Rmd [hunk 6]
```diff
@@ -7,20 +7,50 @@ This event is a key event for professionals in the genomics community interested
...
 
-## Example Schedule {- .unlisted}
+    The report out may be a Google doc, Google slides, a GitHub repo, [a website](https://github.com/jhudsl/OTTR_Template_Website), an [OTTR book](https://github.com/jhudsl/OTTR_Template), etc.
 
-- Problem statement (short demo)
-- Brainstorming
-- Onboarding
-- Work-regroup, work-regroup
-- Report out
+- Share report out during the CoFest Recap Session!
```
Comment (ehumph) [inline organizer-guide.Rmd]: Maybe just "share final report during the CoFest Recap Session!"?

**Review by ehumph — APPROVED** [review]
Overall looks good! Two minor suggestions that just tweak language

**Comment by avahoffman** [comment]
Can we look into why the preview is not rendering correctly?

**Comment by kweav** [comment]
> @kweav sorry, I accidentally committed an edit instead of leaving it a suggestion. Feel free to revert.
> 
> LINE 10 Changed:
> 
> * Build a community (inclusive to newcomers)
> 
> Now reads:
> 
> * Build an inclusive community that is supportive of newcomers
> 
> LINE 12 Changed:
> 
> * Report out a final product that accomplishes one of the following for a topic
> 
> Now reads:
> 
> * Produce a final product that accomplishes one of the following for a topic

These look great! Thank you for making the edits!

**Comment by kweav** [comment]
> Can we look into why the preview is not rendering correctly?

@avahoffman which preview are you referring to -- the quick preview website or comprehensive download? I see the download is a 404 error and it looks like the quick preview website is the un-updated information?

File: organizer-guide.Rmd (hunk 5 above)
Comment (kweav) [reply organizer-guide.Rmd]: Sounds good -- will try to mock one up

File: organizer-guide.Rmd [hunk 7]
```diff
@@ -7,20 +7,50 @@ This event is a key event for professionals in the genomics community interested
...
```
Comment (kweav) [inline organizer-guide.Rmd]:
```suggestion
- Share final report during the CoFest Recap Session!  
```

**Comment by kweav** [comment]
> > Can we look into why the preview is not rendering correctly?
> 
> @avahoffman which preview are you referring to -- the quick preview website or comprehensive download? I see the download is a 404 error and it looks like the quick preview website is the un-updated information?

The quick preview works for me now: https://htmlpreview.github.io/?https://raw.githubusercontent.com/jhudsl/AnVIL_CoFests_2024/preview-9/docs/organizer-guide.html

As does the download

**Comment by ehumph** [comment]
> > > Can we look into why the preview is not rendering correctly?
> > 
> > 
> > @avahoffman which preview are you referring to -- the quick preview website or comprehensive download? I see the download is a 404 error and it looks like the quick preview website is the un-updated information?
> 
> The quick preview works for me now: https://htmlpreview.github.io/?https://raw.githubusercontent.com/jhudsl/AnVIL_CoFests_2024/preview-9/docs/organizer-guide.html
> 
> As does the download

The preview works for me as well (on both my laptop and on my phone).

**Comment by kweav** [comment]
@avahoffman and @ehumph just pushed a potential table setup

**Comment by avahoffman** [comment]
> > Can we look into why the preview is not rendering correctly?
> 
> @avahoffman which preview are you referring to -- the quick preview website or comprehensive download? I see the download is a 404 error and it looks like the quick preview website is the un-updated information?

Yes that's what I was referring to - looks correct now!

**Comment by avahoffman** [comment]
> @avahoffman and @ehumph just pushed a potential table setup

Sweet! Could you push another change? I'm not seeing the table.

**Comment by avahoffman** [comment]
> @avahoffman and @ehumph just pushed a potential table setup

Sweet, thank you! After seeing it, I'm thinking it looks a little cluttered in table form. Maybe we go back to what we had. So sorry for the extra work!

File: organizer-guide.Rmd [hunk 8]
```diff
@@ -7,20 +7,43 @@ This event is a key event for professionals in the genomics community interested
...
-- Build community
-- Develop high interest topics
-- Open science
+- Build an inclusive community that is supportive of newcomers
+- Contribute to [open science](https://en.wikipedia.org/wiki/Open_science)
+- Produce a final product that accomplishes one of the following for a topic
+  - Makes useful forward progress
+  - Identifies opportunities and boundaries or limitations
+
+The final product of CoFests doesn't need to be code! These Collaboration Fests can support story-boarding future work or development of documentation or training materials. If it's something that will help the community and make progress on an AnVIL associated topic and there's sufficient interest, the idea is game! [As Galaxy phrased it for their CoFests:](https://galaxyproject.org/events/gcc2023/cofest/#goals)
```
Comment (avahoffman) [inline organizer-guide.Rmd]:
```suggestion
## Final Product {-}

The final product of CoFests doesn't need to be code! These Collaboration Fests can support story-boarding future work or development of documentation or training materials. If it's something that will help the community and make progress on an AnVIL associated topic and there's sufficient interest, the idea is game! [As Galaxy phrased it for their CoFests:](https://galaxyproject.org/events/gcc2023/cofest/#goals)
```

File: organizer-guide.Rmd [hunk 9]
```diff
@@ -7,20 +7,43 @@ This event is a key event for professionals in the genomics community interested
...
+- 3 Parallel tracks, 5+ hours across Nov 12/13
+  - Scalable Machine Learning and Data Science
+  - Polygenic Risk Score Analysis
+  - Wrap a Tool (+ Intro AnVIL/WDL)
+- Scientific Lead and AnVIL Outreach Co-Lead for support
+- We expect 4-10 participants per track
+
+## Example Schedule for a Track {- .unlisted}
+
+Day 1 (2.5 hrs scheduled)
```
Comment (avahoffman) [inline organizer-guide.Rmd]:
```suggestion
### Day 1 (2.5 hrs scheduled)
```

File: organizer-guide.Rmd [hunk 10]
```diff
@@ -7,20 +7,43 @@ This event is a key event for professionals in the genomics community interested
...
+
+| Task                      | Suggested time | Lead                  | Description |
+|:-------------------------:|:---------------|:----------------------|:------------|
+| Problem statement         | 20-30 minutes  | Scientific Lead       | Provide background info on the track's overall topic and may include a brief summary of the pre-worked collaboration ideas.|
+| Brainstorming             | 15-30 minutes  | Both                  | What project will the track tackle? Focus on understanding the participant backgrounds and skills and matching those with track ideas and actionable tasks to accomplish the goal. |
+| Onboarding                | 30-60 minutes  | AnVIL Outreach Co-lead| Handle billing accounts and other logistics necessary to accomplish the work |
+| Work-regroup, work-regroup| 30-60 minutes  | --                    | Formal collaboration work time left before dinner. (Groups may choose to work together informally after dinner too) |
+
+Day 2 (3 hrs scheduled)
```
Comment (avahoffman) [inline organizer-guide.Rmd]:
```suggestion
### Day 2 (3 hrs scheduled)
```

File: organizer-guide.Rmd [hunk 11]
```diff
@@ -7,20 +7,50 @@ This event is a key event for professionals in the genomics community interested
 
 ## CoFests! Goals {- .unlisted}
 
-- Build community
-- Develop high interest topics
-- Open science
+- Build an inclusive community that is supportive of newcomers
+- Contribute to [open science](https://en.wikipedia.org/wiki/Open_science)
```
Comment (javibio-git) [inline organizer-guide.Rmd]:
```suggestion
- Contribute to [open science](https://en.wikipedia.org/wiki/Open_science) for open collaboration. 
```
Could a main goal of Cofests be the collaborative experience? So maybe specifying one of the key aspects of open science could be necessary.

File: organizer-guide.Rmd (hunk 5 above)
Comment (javibio-git) [reply organizer-guide.Rmd]: I also like the table idea

File: organizer-guide.Rmd [hunk 12]
```diff
@@ -7,20 +7,50 @@ This event is a key event for professionals in the genomics community interested
...
+  - Onboarding by AnVIL co-lead
+
+    Handle billing accounts and other logistics necessary to accomplish the work.
+
+  - Work-regroup, work-regroup
+
+    Probably half an hour to an hour of formal work time left on this first day before dinner. Groups may choose to work together, informally after dinner.
+
+- Second Day (3 hrs scheduled):
+  - Work-regroup, work-regroup
```
Comment (javibio-git) [inline organizer-guide.Rmd]:
```suggestion
  - Work-regroup
```

File: organizer-guide.Rmd [hunk 13]
```diff
@@ -7,20 +7,43 @@ This event is a key event for professionals in the genomics community interested
...
+| Brainstorming             | 15-30 minutes  | Both                  | What project will the track tackle? Focus on understanding the participant backgrounds and skills and matching those with track ideas and actionable tasks to accomplish the goal. |
+| Onboarding                | 30-60 minutes  | AnVIL Outreach Co-lead| Handle billing accounts and other logistics necessary to accomplish the work |
+| Work-regroup, work-regroup| 30-60 minutes  | --                    | Formal collaboration work time left before dinner. (Groups may choose to work together informally after dinner too) |
+
+Day 2 (3 hrs scheduled)
 
-## Example Schedule {- .unlisted}
+| Task                      | Suggested time | Lead                  | Description |
+|:-------------------------:|:---------------|:----------------------|:------------|
+| Work-regroup, work-regroup| 2 - 2.5 hrs    | --                    | Formal collaboration work time (with a coffee break) |
```
Comment (javibio-git) [inline organizer-guide.Rmd]:
```suggestion
| Work-regroup | 2 - 2.5 hrs    | --                    | Formal collaboration work time (with a coffee break) |
```

Or is "Work-regroup, work-regroup" actually meant to be?

Comment (kweav) [reply organizer-guide.Rmd]: It's a phrase that @cutsort used in our meeting and the original draft and I kept it

**Comment by kweav** [comment]
> Minor changes suggested! I think a table would look nice - maybe a table for day 1 and a separate table for day 2?
> 
> Also need to address the spelling items in the dictionary.txt file.

@avahoffman spell check should be addressed as of [commit 2c24440](https://github.com/jhudsl/AnVIL_CoFests_2024/pull/9/commits/2c24440335d873d2dc3416e8f3e93c99ee66d880)

**Comment by kweav** [comment]
> > @avahoffman and @ehumph just pushed a potential table setup
> 
> Sweet, thank you! After seeing it, I'm thinking it looks a little cluttered in table form. Maybe we go back to what we had. So sorry for the extra work!

I really like the edits to the table you made in PR #11 so leaving them in this one as is, knowing they've been addressed there -- thank you!

File: organizer-guide.Rmd [hunk 14]
```diff
@@ -7,20 +7,45 @@ This event is a key event for professionals in the genomics community interested
 
 ## CoFests! Goals {- .unlisted}
 
-- Build community
-- Develop high interest topics
-- Open science
+- Build an inclusive community that is supportive of newcomers
+- Contribute to [open science](https://en.wikipedia.org/wiki/Open_science)
```
Comment (avahoffman) [inline organizer-guide.Rmd]:
```suggestion
- Contribute to [open science](https://en.wikipedia.org/wiki/Open_science) and collaboration
```

File: organizer-guide.Rmd (hunk 11 above)
Comment (kweav) [reply organizer-guide.Rmd]: Ah, I really like the of pointing out collaboration is a key aspect of open science! I'll slightly rephrase this but very grateful for the suggestion and stressing the importance of collaboration. Thanks!

File: organizer-guide.Rmd (hunk 14 above)
Comment (kweav) [inline organizer-guide.Rmd]:
```suggestion
- Collaborate and contribute to [open science](https://en.wikipedia.org/wiki/Open_science)
```

**Pull request merged**